spark/inspections/CometTPCHQueriesList-results.txt

```diff
@@ -1,142 +1,129 @@
 Query: q1 TPCH Snappy. Comet Exec: Enabled (CometHashAggregate, CometFilter, CometProject)
 Query: q1 TPCH Snappy: ExplainInfo:
-Shuffle: unsupported Spark partitioning: org.apache.spark.sql.catalyst.plans.physical.RangePartitioning
+Comet shuffle is not enabled: spark.sql.adaptive.coalescePartitions.enabled is enabled and spark.comet.shuffle.enforceMode.enabled is not enabled
```

Comment:
The changes to explain info occur because Spark generates different plans at different scales (especially wrt to broadcast). 
Let me run with this PR and get an updated version.

Reply:
@advancedxy, here you go - https://github.com/parthchandra/datafusion-comet/commit/95443622c225b7d955bba6261471eed5f82b92b7.patch